src/skadi/util.cc

```diff
@@ -71,7 +76,8 @@ weighted_grade(const std::vector<double>& heights,
   // Get the average weighted grade by homogenizing total weight. Return
   // max grades (up and down) and mean elevation.
   return std::make_tuple(total_grade * (1.0 / total_weight), max_up_grade, max_down_grade,
-                         (total_elev / n));
+                         (has_only_no_data ? static_cast<double>(get_no_data_value())
+                                           : total_elev / n));
```

Comment:
its already a double

```suggestion
                         has_only_no_data ? get_no_data_value() : total_elev / n));
```

Reply:
:facepalm: of course we can.. changed it.